### DISCUSSION TITLE
is vcpkg.io associated with this repository and how often does it's index update

### DISCUSSION THREAD

**Discussion by braindigitalis** [microsoft/vcpkg#26622] [Q&A] [answered]
the website vcpkg.io doesn't seem to be updated with packages added to vcpkg 6 months or more ago.

is it operated by the maintainers of this repository, and how often does it update?

do updates have to be manually performed?

thanks!

**Comment by autoantwort** [comment, marked as answer]
> do updates have to be manually performed?

Yes

**Comment by TLescoatTFX** [comment]
Hello, vcpkg.io seems out-of-date, is it still a manual deployment or is there some automation ? If so, at what frequency ?
Thanks
